### PR TITLE
Add inventory welcome screen

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: webpack/InsightsCloudSync/InsightsCloudSync.js [hunk 1]
```diff
@@ -1,26 +1,70 @@
...
-        </Button>
+        <p>
+          {__(`Insights synchronization process is used to provide Insights
+           recommendations output for hosts managed here`)}
+        </p>
+        <p>
+          <FormattedMessage
+            id="rh_cloud.insights.token_hint"
+            description="Hint where to obtain API token"
+            defaultMessage={`In order to set up the sychronization, you should obtain an RHSM API token
```
Comment (laviro) [inline webpack/InsightsCloudSync/InsightsCloudSync.js:22]: should this string also be translated?

File: webpack/InsightsCloudSync/InsightsCloudSync.js [hunk 2]
```diff
@@ -1,26 +1,70 @@
...
+              from {apiLink}`}
+            values={{
+              apiLink: (
+                <a
+                  href="https://access.redhat.com/management/api"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  access.redhat.com
+                  <i className="fas fa-external-link-alt" />
```
Comment (laviro) [inline webpack/InsightsCloudSync/InsightsCloudSync.js:32]: can you use the `Patternfly Icon` component for it? e.g: `<Icon name="external-link" />`
and can you make it a bit smaller?

**Comment by ShimShtein** [comment]
@laviro, I think it's ready for another round

File: webpack/InsightsCloudSync/InsightsCloudSync.js [hunk 3]
```diff
@@ -2,25 +2,52 @@ import React from 'react';
...
+        </p>
+        <p>
+          {__(`1. Obtain an RHSM API token: `)}
+          <a
+            href="https://access.redhat.com/management/api"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            access.redhat.com
+            <Icon name="external-link" />
```
Comment (laviro) [inline webpack/InsightsCloudSync/InsightsCloudSync.js:26]: can you add size="xs" to the Icon? and also add a space just before it?

File: webpack/InsightsCloudSync/InsightsCloudSync.js [hunk 4]
```diff
@@ -2,25 +2,52 @@ import React from 'react';
...
+          </a>
+          <br />
+          {__("2. Copy the token to 'Red Hat Cloud token' setting: ")}
+          <a href={settingsUrl}>{__('Red Hat Cloud token')}</a>
```
Comment (laviro) [inline webpack/InsightsCloudSync/InsightsCloudSync.js:30]: can we also open this link in another tab?
it can take time to find the right page again.

**Pull request closed without merging**